src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -810,7 +809,11 @@ private class FastqDemultiplexer(val sampleInfos: Seq[SampleInfo],
 
     val passQc = demuxRecords.forall(d => d.readInfo.forall(_.passQc))
     val isControl = demuxRecords.forall(d => d.readInfo.forall(_.internalControl))
-    DemuxResult(sampleInfo=sampleInfo, numMismatches=numMismatches, records=demuxRecords, passQc=passQc, isControl=isControl)
+
+    val result = DemuxResult(sampleInfo=sampleInfo, numMismatches=numMismatches, records=demuxRecords, passQc=passQc, isControl=isControl)
+    result.sampleInfo.metric.increment(numMismatches=numMismatches, isPf=passQc)
```

Comment:
Is this the right place for this?  Suppose I am omitting control reads, then they will be counted here.  I think we don't want them counted.  I think you want to increment in `demultiplexingIterator`?